src/src/DataStructs/DeviceStruct.h

```diff
@@ -92,6 +92,7 @@ struct __attribute__((__packed__)) DeviceStruct
   bool PullUpOption       : 1;       // Allow to set internal pull-up resistors.
   bool InverseLogicOption : 1;       // Allow to invert the boolean state (e.g. a switch)
   bool FormulaOption      : 1;       // Allow to enter a formula to convert values during read. (not possible with Custom enabled)
+  bool easyfetchOption    : 1;       // Allow to enter optional easyfetch values

```

Comment:
Could have chosen a full camel-case name `EasyFetchOption`, like all the other Device flags. And I'm not sure if Option should be part of that name 🤔 (could maybe be `EasyFetchArgs` or `EasyFetchConfig` ?)

Reply:
EasyFetchArgs sounds good to me